rbx_types/src/tags.rs

```diff
@@ -0,0 +1,108 @@
+use std::convert::TryFrom;
+use std::str::FromStr;
+use std::string::FromUtf8Error;
+
+#[derive(Default, Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(transparent)
+)]
+pub struct Tags {
+    tag_list: Vec<String>,
+}
+
+impl Tags {
+    pub fn new() -> Tags {
+        Self {
+            tag_list: Vec::new(),
+        }
+    }
+
+    pub fn as_vec(&self) -> &Vec<String> {
+        &self.tag_list
+    }
```

Comment:
You never want to return `&Vec<T>` in Rust. How about `fn as_slice(&self) -> &[String]`?